ci-operator/step-registry/hypershift/mce/power/create/nodepool/hypershift-mce-power-create-nodepool-commands.sh

```diff
@@ -70,188 +65,45 @@ export IBMCLOUD_TRACE=true
 echo "$(date) Creating VSI in PowerVS instance"
 ibmcloud pi ins create ${POWERVS_VSI_NAME} --image ${POWERVS_IMAGE} --subnets ${POWERVS_NETWORK} --memory ${POWERVS_VSI_MEMORY} --processors ${POWERVS_VSI_PROCESSORS} --processor-type ${POWERVS_VSI_PROC_TYPE} --sys-type ${POWERVS_VSI_SYS_TYPE} --replicants ${HYPERSHIFT_NODE_COUNT} --replicant-scheme suffix --replicant-affinity-policy affinity
 
-# Applying mirror config
-echo "$(date) Applying mirror config"
-cat <<EOF | oc apply -f -
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: mirror-config
-  namespace: multicluster-engine
-  labels:
-    app: assisted-service
-data:
-  registries.conf: |
-    unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
-
-    [[registry]]
-      location = "registry.stage.redhat.io"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io"
-        insecure = false
-
-    [[registry]]
-      location = "registry.redhat.io"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io"
-        insecure = false
-
-    [[registry]]
-      location = "registry.redhat.io/rhacm2"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io/rhacm2"
-        insecure = false
-
-    [[registry]]
-      location = "registry-proxy.engineering.redhat.com"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io"
-        insecure = false
-
-    [[registry]]
-      location = "registry.redhat.io/multicluster-engine"
-      insecure = false
-      blocked = false
-      mirror-by-digest-only = true
-      prefix = ""
-
-      [[registry.mirror]]
-        location = "brew.registry.redhat.io/multicluster-engine"
-        insecure = false
-EOF
-
+MCE_VERSION=$(oc get "$(oc get multiclusterengines -oname)" -ojsonpath="{.status.currentVersion}" | cut -c 1-3)
+ARCH="ppc64le"
+if (( $(echo "$MCE_VERSION < 2.6" | bc -l) )); then
+ # Support for using ppc64le arch added after MCE 2.6
+ ARCH="amd64"
+fi
```

Comment:
Looking ahead, I think we can define `HYPERSHIFT_NODE_ARCH` or `ADDITIONAL_HYPERSHIFT_NODE_ARCH` in the step, which will make it more intuitive to see the architecture of the nodepool.
https://github.com/openshift/release/blob/db53a52bad3033a5bde403ea64745f5acfffc19c/ci-operator/step-registry/hypershift/aws/create/nodepool/hypershift-aws-create-nodepool-ref.yaml#L12

Reply:
I feel in this context, this will not make much value, since it comes under mce-**power**-create step, obviously its going to use ppc64le arch, where as in aws case, it can have different arch so it would make sense there.
Also when we start working on heterogeneous workflow where multiple arch would be involved, we can't use the common `HYPERSHIFT_NODE_ARCH` across different arch nodepool creation steps.

Reply:
Okay, Thanks